core/native/src/main/scala/zio/PlatformSpecific.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2017-2020 John A. De Goes and the ZIO Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package zio
+
+import zio.clock.Clock
+import zio.console.Console
+import zio.random.Random
+import zio.scheduler.Scheduler
+import zio.system.System
+
+private[zio] trait PlatformSpecific {
+  type ZEnv = Clock with Console with System with Random with Scheduler
+
+  private[zio] val defaultEnvironment: Managed[Nothing, ZEnv] =
```

Comment:
This should actually return `ZLayer` for consistency with the other ones.

Reply:
Okay. The current signature is consistent across all three platform specific files currently so I will change all of them.

Reply:
Ah I see you already fixed this. Will incorporate your changes.